src/main/java/com/onarandombox/MultiverseCore/utils/WorldManager.java

```diff
@@ -403,6 +418,22 @@ public boolean loadWorld(String name) {
         }
     }
 
+    /**
+     * Check if the world name is allowed
+     *
+     * @param name   Name of the world
+     * @return True if the world world name is valid based on regex
+     */
+    private boolean isValidWorldName(String name) {
+        String worldNamePattern = "[a-zA-Z0-9/._-]+";
+        if (!name.matches(worldNamePattern)) {
```

Comment:
I don't see where you were using the imported `Pattern` class but I do tend to prefer to use `Pattern.compile` and store that in a static var. I assume it's faster but I'm not sure I actually know that. :)